utils/openebs-cleanup.yml

```diff
@@ -53,31 +53,15 @@
   retries: 30
   delay: 5
 
-- name: Check openebs is removed or not from cluster
+- name: Checking for openebses status after removing openebs
```

Comment:
I think it's better to have this as a negative check i.e. since we have removed the OpenEBS entry in the previous steps, this check should only check that this cluster should have no active OpenEBS entry and in the next step we should always create a new OpenEBS entry for this cluster and this step will be executed only if no active OpenEBS entry is present.